### ISSUE TITLE
Replacing the Avatar App icon

### ISSUE THREAD

**Issue by BLKZOR** [overte-org/overte#1109]
The current Avatar App icon is very similar to the icons for accessibility features used in many applications, which can be confusing.
I believe that the current icon should be reserved for accessibility features and that a new Avatar App icon needs to be discussed.

I propose the icon shown in the attached diagram:
![image](https://github.com/user-attachments/assets/c906ee5e-07cc-4a2b-9463-2b8f68fd7c12)
which represents the Dressing room, but which I believe best describes the functionality of the Avatar App.

I would be happy to hear any other opinions or suggestions.

**Comment by ksuprynowicz** [comment]
That's a good point. Current icon is indeed similar to accesibility icon. I wonder what other icons we could use for avatar app?

**Comment by JulianGro** [comment]
I quite like the current avatar app icon, but adding a clothes hanger is indeed a very good idea.
Where did you get that icon, which you proposed, from?
I think just the current avatar app icon moved a tiny bit to the side with a clothes hanger attached to it would already be great. The current icons (avatar-i.svg and avatar-a.svg) are at https://github.com/overte-org/overte/tree/master/interface/resources/icons/tablet-icons

**Comment by Armored-Dragon** [comment]
I think adding an additional icon next to the existing avatar icon would make the toolbar button a bit too cluttered. How about replacing the existing avatar icon with just a hanger? We might be able to completely repurpose the current avatar icon into the accessibility icon.

**Comment by ksuprynowicz** [comment]
Maybe just person part of the icon instead, or person + coathanger proposed by BLKZOR?

**Comment by JulianGro** [comment]
> Could we not have coat hanger as a symbol for avatar app?

Just a coat hanger, or a person with a small coat hanger as well? (I don't necessarily understand your aversion towards coat hangers, even if I also wouldn't like an icon that is only a coat hanger for the avatar app.)

**Comment by JulianGro** [comment]
Some suggestions:
- Current icon holding a hoat hanger.
- Current icon with its arms lowered (but not all the way).
- sitelen pona for sweet person: (jan suwi) 
![jOzB1yZrIwtvDn-g23vWoCG6sKuRhQceGL96_t_Uyfc_jan_suwi](https://github.com/user-attachments/assets/e9f6b3ff-5178-429d-a1b7-01b0d8ec9c90) (Not sure if something like this would look good, but it could be cute to use something like that. Like, what could be cuter than sweet human written in sitelen pona.)

**Comment by ksuprynowicz** [comment]
> Just a coat hanger, or a person with a small coat hanger as well? (I don't necessarily understand your aversion towards coat hangers, even if I also wouldn't like an icon that is only a coat hanger for the avatar app.)

Just the coat hanger. Person + small coat hanger is ok.

**Comment by ksuprynowicz** [comment]
>     * sitelen pona for sweet person: (jan suwi)

I absolutely love this idea.

**Comment by BLKZOR** [comment]
>  Where did you get that icon, which you proposed, from?

For this proposal I have created it in Inkscape.
![avatar app icon](https://github.com/user-attachments/assets/de884f5c-5f83-4c7d-b19a-4e1e6586e822)
Attached is an svg. Variations in the shape of the hanger have also been added.

Anything added to the current Avatar App icon should be avoided. It is still reminiscent of accessibility features.
For example, the Microsoft 365 Accessibility Checker rates accessibility with a "✓", "✗", or "?", like this:
https://github.com/overte-org/overte/blob/master/interface/resources/icons/tablet-icons/ignore.svg

Hanger-only icons were also considered as candidates. cluster is a real example.

I'll add one more idea that doesn't involve hanger:
![image](https://github.com/user-attachments/assets/98a7e326-b5a1-4c57-8971-2ad96f56da20)
![avatar app icon_2](https://github.com/user-attachments/assets/23f86fcc-2a19-48ad-b13c-916bc7dba469)

**Comment by JulianGro** [comment]
Alezia made a vote here:
https://framadate.org/ZysIImpd1Rc8zyd9
(End of vote: September 18th, 2024, EOD)
![ICON_VOTE](https://github.com/user-attachments/assets/2e7e41c7-a783-4f1a-9c74-27a31761d7d6)

**Comment by AleziaKurdis** [comment]
Icon no 3 is officially the new Avatar App icon.
![image](https://github.com/user-attachments/assets/03eef150-c33e-404f-823a-5313882f7199)

**Comment by AleziaKurdis** [comment]
Here are the new icons for the replacement. (active and inactive)
I reprocessed it to get a cleaner svg, and created the white version.
[new icons (a and i).zip](https://github.com/user-attachments/files/17052418/new.icons.a.and.i.zip)

They should go there: 
**overte/interface/resources/icons/tablet-icons/**
Who want to do a PR?
  
----
That said, it is also used a bit everywhere in the app it-self:
![image](https://github.com/user-attachments/assets/899a345d-f6a3-403e-ae37-d01ebbfe025b)
I suspect those to not be a .svg but a font character (part of the hifi glyph font)
If it is the case, I can help with this, but I would suggest to open a new font (overte_glyph) if possible to avoid to edit the hifi one (this is certainly easy to mess.)
